ansible/roles/test/files/saitests/pfc_asym.py

```diff
@@ -32,7 +32,7 @@ class PfcAsymBaseTest(ThriftInterfaceDataPlane):
     PACKET_ECN = 1
     PACKET_TTL = 64
     PACKET_LEN = 72
-    PACKET_NUM = 10000
+    PACKET_NUM = 100000
```

Comment:
Why increase this 10x times? Will this significantly increase the time required for running this test?

Reply:
This almost will not increase execution time. It was increased for adding more stability when DUT has high speed ports like 100G and port buffer is not yet overloaded by PFC frames.

Reply:
OK